server/sonar-server/src/test/java/org/sonar/server/computation/task/projectanalysis/period/PeriodsHolderRule.java

```diff
@@ -46,12 +46,22 @@ private void clear() {
     this.delegate = new PeriodsHolderImpl();
   }
 
+  /**
+   * @deprecated as only one period is now available. Use {@link #setPeriod(Period)} instead
+   */
+  @Deprecated
   public PeriodsHolderRule setPeriods(Period... periods) {
     delegate = new PeriodsHolderImpl();
     delegate.setPeriods(Arrays.asList(periods));
     return this;
   }
 
+  public PeriodsHolderRule setPeriod(Period period) {
```

Comment:
The leak is a period, there's only one period, for the moment I think it's ok.
This could be easily done later